docs/reference/search/search-your-data/paginate-search-results.asciidoc

```diff
@@ -71,9 +71,13 @@ To get the first page of results, submit a search request with a `sort`
 argument. If using a PIT, specify the PIT ID in the `pit.id` parameter and omit
 the target data stream or index from the request path.
 
-IMPORTANT: We recommend you include a tiebreaker field in your `sort`. This
-tiebreaker field should contain a unique value for each document. If you don't
-include a tiebreaker field, your paged results could miss or duplicate hits.
+IMPORTANT: If you cannot use a PIT, we recommend that you include a tiebreaker field
```

Comment:
may be we need to add  a more clear note:  All PIT search requests add an implicit tiebreak field called `_shard_doc`.

Reply:
++, I pushed https://github.com/elastic/elasticsearch/pull/68833/commits/2591295ea9a7a07a21b44ec26404d0ca129e0359